plaidml/op/op_test.cc

```diff
@@ -196,6 +196,15 @@ TEST_F(OpTest, Convolution) {
   runProgram(program);
 }
 
+TEST_F(OpTest, Convolution_Padding3) {
```

Comment:
I was trying to move our tests to something a little more parameterized (like OpenVINO tests):
https://github.com/plaidml/plaidml/pull/1221